esp/templates/program/modules/adminclass/manageclass_class_info.html

```diff
@@ -1,6 +1,7 @@
 <tr>
     <th class="smaller">Teachers:</th>
-    <td>{{ class.getTeacherNames|join:", " }}<br />
+	<td>{% for teacher in class.get_teachers %}<a href="/manage/userview?username={{ teacher.username|urlencode }}">{{ teacher.nonblank_name }}</a>{% if not forloop.last %}, {% endif %}{% endfor %}
```

Comment:
The correct way to do this definitely involves `{% url something %}`, but I really want these links for sub wrangling and the rest of the codebase uses this URL creation method *a lot* so I'm going to keep this until after Splash at least